mutable_tree.go

```
@@ -73,6 +73,14 @@ func (tree *MutableTree) IsEmpty() bool {
 
 // VersionExists returns whether or not a version exists.
 func (tree *MutableTree) VersionExists(version int64) bool {
+	legacyLatestVersion, err := tree.ndb.getLegacyLatestVersion()
+	if err != nil {
+		return false
+	}
+	if version <= legacyLatestVersion {
+		has, _ := tree.ndb.hasLegacyVersion(version)
+		return has
```

If ignoring the error is ok, it's clearer to do something like:

```
		has, err := tree.ndb.hasLegacyVersion(version)
		return err == nil && has
```